landoapi/models/treestatus.py

```diff
@@ -6,25 +6,63 @@
 This module provides the definitions for Treestatus data.
 """
 
+import enum
 import json
-
 from typing import (
     Any,
     Optional,
 )
 
 import sqlalchemy.ext.hybrid
-
 from sqlalchemy.orm import (
     relationship,
 )
 
 from landoapi.models.base import (
-    db,
     Base,
+    db,
 )
 
-DEFAULT_TREE = {"reason": "New tree", "status": "closed", "tags": [], "log_id": None}
+
+class TreeCategory(enum.Enum):
+    """Categories of the various trees.
+
+    Note: the definition order is in order of importance for display in the UI.
+    Note: this class also exists in Lando-UI, and should be updated in both places.
```

Comment:
I doubt this will change between the Treestatus migration and our UI/API consolidation with the Django migration. I'm going to leave it as-is and the duplication will disappear once that work is completed.